Sources/App/Migrations.swift

```diff
@@ -71,8 +74,5 @@ class Migrations {
         } catch {
```

Comment:
Migration is now handled using the Procfile as recommended for Production on release, so autoMigrate should only be required for development.

Reply:
Given we only added that in due to a mistake, I'd rather we leave it how we had it than to chop out what we already had and risk further changes to the migrations and database consistency.

I did peep through the [docs](https://docs.vapor.codes/deploy/heroku/#create-an-application) after your previous change, and I don't believe there's any explicit recommendation on how to handle this, only a sample showing that you _can_ run them via a Heroku command.